dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/SecurityOwnershipValidator.java

```diff
@@ -304,9 +300,7 @@ private void checkEventOrgUnitWriteAccess(
       OrganisationUnit eventOrgUnit,
       boolean isCreatableInSearchScope,
       UserDetails user) {
-    if (eventOrgUnit == null) {
-      log.warn(ORG_UNIT_NO_USER_ASSIGNED, event.getUid());
-    } else if (isCreatableInSearchScope
+    if (isCreatableInSearchScope
```

Comment:
maybe I'm missing something...
but [here](https://github.com/dhis2/dhis2-core/pull/19193/files#diff-f19b9c82388c030b253a1dc12bdd536b1ac900bb398162a5aca6bd81c9cfb894R99) we validate the user capture scope if the strategy is not an UPDATE.
since this method is only called when the strategy is not update, do we need to validate the org unit scope again?

Reply:
This whole validator is quite messy and difficult to follow.
We already have a bug to double check that everything in here is correct.
https://dhis2.atlassian.net/browse/DHIS2-17335